lib/auth/clt.go

```diff
@@ -1065,4 +1065,7 @@ type ClientI interface {
 	// and prompts the user to answer the challenge with the given promptOpts, and ultimately returning
 	// an MFA challenge response for the user.
 	PerformMFACeremony(ctx context.Context, challengeRequest *proto.CreateAuthenticateChallengeRequest, promptOpts ...mfa.PromptOpt) (*proto.MFAAuthenticateResponse, error)
+
+	// GetClusterAccessGraphConfig retrieves the Cluster Access Graph configuration from Auth server.
```

Comment:
minor nit: I think we only capitalize product and component (Auth, Proxy) names.
```suggestion
	// GetClusterAccessGraphConfig retrieves the cluster Access Graph configuration from Auth server.
```